.github/actions/oblt-cli-create-serverless/action.yml

```diff
@@ -0,0 +1,107 @@
+name: 'Oblt-cli create serverless'
+description: 'Run the oblt-cli wrapper to create a serverless cluster.'
+inputs:
+  cluster-name-prefix:
+    description: 'Prefix to be prepended to the randomised cluster name'
+    required: false
+  cluster-name-suffix:
+    description: 'Suffix to be appended to the randomised cluster name'
+    required: false
+  target:
+    description: 'The target environment where to deploy the serverless cluster. Default: `qa`'
+    default: "qa"
+    required: false
+  project-type:
+    description: 'The project type. Default: `observability`'
+    default: "observability"
+    required: false
+  token:
+    description: 'The GitHub access token.'
+    required: true
+  slackChannel:
+    description: 'The slack channel to notify the status.'
+    default: '#observablt-bots'
+    required: false
+  username:
+    description: 'Username to show in the deployments with oblt-cli, format: [a-z0-9]'
+    default: 'apmmachine'
+    required: false
+  gitops:
+    description: 'Whether to provide the GitOps metadata to the oblt-cli'
+    default: false
+    required: false
+  dry-run:
+    description: 'Whether to dryRun'
+    default: false
+    required: false
+runs:
+  using: "composite"
+  steps:
+
+    - name: Configure oblt-cli flags
+      run: |
+        {
+          [ -n '${{ inputs.cluster-name-prefix }}' ] \
+            && echo -n '--cluster-name-prefix=${{ inputs.cluster-name-prefix }} '
+          [ -n '${{ inputs.cluster-name-suffix }}' ] \
+            && echo -n '--cluster-name-suffix=${{ inputs.cluster-name-suffix }} '
+          [ '${{ inputs.dry-run }}' == 'true' ] \
+            && echo -n '--dry-run '
+        } > .flags
+        echo "COMMAND=$(cat .flags)" >> $GITHUB_ENV
+        rm .flags
```

Comment:
NIT: this in action script is more clear and about the same length of code.

Reply:
Let me revisit this

Reply:
I'll create a follow up